gridstatus/caiso.py

```diff
@@ -1484,10 +1496,17 @@ def _get_historical(
     # but has nulls for all other columns
     df = df.dropna(subset=df.columns[1:], how="all")
 
+    # sometimes this data is from the previous day, in which case we want to label it as such
+    latest_file_time = utils.check_latest_value_time(df, file)
+    current_caiso_time = pd.Timestamp.now(tz=CAISO.default_timezone)
+
+    if latest_file_time > current_caiso_time:
```

Comment:
the tests are failing because the logic here is flawed. 

we only want to do this check if we are querying for the non-dated file.

Reply:
added logic to check only if it's the non-dated.